pkg/image/export/main.go

```diff
@@ -88,6 +86,9 @@ func run(images ...string) error {
 		targetImages = append(targetImages, image.Mirror(i))
 	}
 
+	k8sVersionToWindowsSystemImages, _ := kd.GetK8sVersionInfo(rancherVersion,
```

Comment:
is there some way to combine this for both linux and windows?  Actual logic of calculating k8s versions is happening twice. This function gets called only by `pkg/image/export/main.go` so feel free to make changes as you like :-)

Reply:
Updated